imagersite/imager_profile/tests.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import unicode_literals
+from django.test import TestCase
+import factory
+from django.contrib.auth.models import User
+from imager_profile.models import ImagerProfile
+from faker import Faker as fake_factory
+
+fake = fake_factory()
+
+
+class UserFactory(factory.Factory):
+    class Meta:
+        model = User
+    first_name = fake.first_name()
+    last_name = fake.last_name()
+    email = factory.LazyAttribute(lambda x:
+                                  '{0}@example.com'.format(x.first_name))
+    username = factory.Sequence(lambda n: 'user{}'.format(n))
+
+
+class ProfileTestCase(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(ProfileTestCase, cls).setUpClass()
+        for i in range(100):
+            user = UserFactory.create()
+            user.save()
+
+    @classmethod
+    def tearDownClass(cls):
+        User.objects.all().delete()
+        super(ProfileTestCase, cls).tearDownClass()
+
+    def test_profile_is_created_when_user_is_saved(self):
+        self.assertTrue(ImagerProfile.objects.count() == 100)
```

Comment:
It would be more demonstrative of the functionality to first assert there are none, then create a new user, and show that there is one.  This would prove the point that it is saving the user that creates the profile object.